src/distilabel/utils/logging.py

```diff
@@ -57,6 +59,11 @@ def setup_logging(log_queue: "Queue[Any]") -> None:
         handler.setFormatter(formatter)
         queue_listener = QueueListener(log_queue, handler, respect_handler_level=True)
         queue_listener.start()
+        # Given that we always append to the log file, remove it from the main process,
+        # this way we ensure every pipeline starts with a fresh log file.
+        log_filename = Path(filename)
+        if log_filename.exists():
+            log_filename.unlink()
```

Comment:
I think is better to have the default append logic, right? It will also cover the case you execute the pipeline, you stop it, then you execute it again and continue from the point it was stopped, and you have the log messages from the two executions that produced the final dataset.